source/making_payments/index.html.md.erb

```diff
@@ -54,15 +61,19 @@ For example:
 }
 ```
 
-### API call parameters
+Depending on your integration, you can also:
 
-You can see definitions and possible values of every parameter and attribute for this endpoint in our [API reference](/api_reference/create_a_payment_reference).
+* [delay taking a payment](/delayed_capture)
+* [prefill some of the fields on the user's payment page](/optional_features/prefill_user_details/)
+* [add custom metadata to a payment](/custom_metadata/)
+* [use Welsh on your payment pages](/optional_features/welsh_language/)
+* [take the payment over the phone](/moto_payments)
 
-#### amount
+You can see definitions, values, and the limits of every parameter for this endpoint in our [API reference](/api_reference/create_a_payment_reference).
 
-`amount` is the amount in pence. In the example, the payment is for £145.
+#### amount
 
-The amount must be a number data type rather than a string.
+The `amount` parameter sets the payment amount in pence.
```

Comment:
Do we state it somewhere that amount must be of number type? It's a very important piece of information / rule

Reply:
I've added this now.